src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -152,26 +158,3 @@ export default class ResultingBusinessName extends Mixins(AmalgamationMixin, Nam
   }
 }
 </script>
-
-<style lang="scss" scoped>
-// position the Undo button "on top of" NameRequestInfo
-.btn-undo {
-  position: absolute;
-  top: 22px;
-  right: 20px;
-}
-
-// "sm" breakpoint
-@media (min-width: 600px) {
-  .btn-undo {
-    top: 24px;
-  }
-}
-
-// "md" breakpoint
-@media (min-width: 960px) {
-  .btn-undo {
-    top: 28px;
-  }
-}
-</style>
```

Comment:
What does the Undo button look like now on a regular amalgamation ?

Reply:
This is what's on dev:
![image](https://github.com/bcgov/business-create-ui/assets/17242795/19ba1220-5692-4c69-bf86-1b0b777e125b)

Reply:
No, what do you see (locally) after deleting this CSS?

Reply:
I'll put that CSS back. It's supposed to still be there. Removing the CSS moves "Undo" to the bottom of the container.

Reply:
Reverted, thanks. Resolving.